pkgs/desktops/gnome-3/update.nix

```diff
@@ -1,16 +1,24 @@
-{ stdenv, lib, writeScript, python3, common-updater-scripts }:
-{ packageName, attrPath ? packageName, versionPolicy ? "odd-unstable" }:
+{ stdenv, pkgs, lib, writeScript, python3, common-updater-scripts }:
+{ packageName, attrPath ? packageName, versionPolicy ? "odd-unstable", freeze ? true }:
 
 let
   python = python3.withPackages (p: [ p.requests ]);
+  upperBoundFlag =
+    let
+      package = lib.getAttrFromPath (lib.splitString "." attrPath) pkgs;
+      minorVersion = lib.versions.minor package.version;
```

Comment:
If I ran update.nix just on `gnome3` attribute I got
```
error: attribute 'version' missing
```

I guess it's possible some packages may not have a `version` attribute.
Maybe use `builtins.parseDrvName` to get `version` from `name`?

Reply:
```suggestion
      minorVersion = lib.versions.minor (builtins.parseDrvName package.name).version;
```